src/Components/WebAssembly/DevServer/src/Microsoft.AspNetCore.Components.WebAssembly.DevServer.csproj

```diff
@@ -12,24 +12,10 @@
     <NoWarn>$(NoWarn.Replace('1591', ''))</NoWarn>
     <!-- Set this to false because assemblies should not reference this assembly directly, (except for tests, of course). -->
     <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
-
-    <!--
-      This project compiles against Microsoft.AspNetCore.App from the SDK.
-      This ensures that it's packaging output is correct and does not include local artifacts.
-    -->
-    <UseAspNetCoreSharedRuntime>true</UseAspNetCoreSharedRuntime>
-    <DoNotApplyWorkaroundsToMicrosoftAspNetCoreApp>true</DoNotApplyWorkaroundsToMicrosoftAspNetCoreApp>
   </PropertyGroup>
 
   <ItemGroup>
-    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Why did this reference change? I'm not too sure about the changes in this file but as long as @pranavkm signs off it should be fine.

Reply:
Probably putting an error above that indicating that we expect "netN" so that we catch this location if something changes.

Reply:
The whole project changed because we have a mess of conflicts because this project attempted to use AspNetCore bits from the SDK as well as reference projects elsewhere in the repo. @pranavkm signed off on doing this in Teams.

 #26448 is about trimming the package _correctly_ at some point in the 6.0 train but only if this project doesn't move elsewhere.